benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/query/cassandra/CassandraQueryRunner.java

```diff
@@ -36,15 +52,36 @@ public void runQuery() throws Exception {
    * @param query Query to run against Cassandra database.
    */
   @Override
-  public void prepareQueryRunner(String query) throws Exception {
-
+  public void prepareQueryRunner(String query) {
+    cluster = Cluster.builder()
+            .addContactPoint(NODE)
+            .build();
+    session = cluster.connect();
+    this.query = query;
+    queryWasSuccessful = true;
   }
 
   /**
    * Function to check query execution status against Cassandra database.
    */
   @Override
   public void checkQueryExecutionStatus(String benchmarkPath) throws Exception {
+    session.close();
+    cluster.close();
+
+    if (queryWasSuccessful) {
+      return;
+    }
 
+    File benchmarkDirectory = new File(benchmarkPath);
+    if (benchmarkDirectory.exists() && benchmarkDirectory.isDirectory()) {
+      BufferedWriter bufferedWriter = new BufferedWriter(
+              new FileWriter(benchmarkPath + "/failed_queries.txt", true));
```

Comment:
maybe rename this file & elasticsearch file as `<elasticsearch/cassandra>_failed_queries` if it makes sense

Reply:
Good idea